src/editor/Editor.js

```diff
@@ -59,99 +59,120 @@
-        _                  = require("thirdparty/lodash");
+        Strings = require("strings"),
+        TextRange = require("document/TextRange").TextRange,
+        TokenUtils = require("utils/TokenUtils"),
+        ValidationUtils = require("utils/ValidationUtils"),
+        HTMLUtils = require("language/HTMLUtils"),
+        ViewUtils = require("utils/ViewUtils"),
+        MainViewManager = require("view/MainViewManager"),
+        DocumentManager = require("document/DocumentManager"),
+        FileUtils = require("file/FileUtils"),
+        StringUtils = require("utils/StringUtils"),
+        Dialogs = require("widgets/Dialogs"),
+        DefaultDialogs = require("widgets/DefaultDialogs"),
+        ProjectManager = require("project/ProjectManager"),
+        EditorManager = require("editor/EditorManager"),
+        DocumentCommandHandlers = brackets.getModule("document/DocumentCommandHandlers"),
+        _ = require("thirdparty/lodash"),
+        Db = require("editor/Db"),
+        CompressionUtils = require("thirdparty/rawdeflate"),
+        CompressionUtils = require("thirdparty/rawinflate"),
+        He = require("thirdparty/he");
 
     /** Editor preferences */
 
-    var CLOSE_BRACKETS      = "closeBrackets",
-        CLOSE_TAGS          = "closeTags",
-        DRAG_DROP           = "dragDropText",
-        HIGHLIGHT_MATCHES   = "highlightMatches",
-        LINEWISE_COPY_CUT   = "lineWiseCopyCut",
-        SCROLL_PAST_END     = "scrollPastEnd",
-        SHOW_CURSOR_SELECT  = "showCursorWhenSelecting",
-        SHOW_LINE_NUMBERS   = "showLineNumbers",
-        SMART_INDENT        = "smartIndent",
-        SOFT_TABS           = "softTabs",
-        SPACE_UNITS         = "spaceUnits",
-        STYLE_ACTIVE_LINE   = "styleActiveLine",
-        TAB_SIZE            = "tabSize",
-        UPPERCASE_COLORS    = "uppercaseColors",
-        USE_TAB_CHAR        = "useTabChar",
-        WORD_WRAP           = "wordWrap",
-        AUTO_HIDE_SEARCH    = "autoHideSearch",
-        INDENT_LINE_COMMENT   = "indentLineComment",
+    var CLOSE_BRACKETS = "closeBrackets",
+        CLOSE_TAGS = "closeTags",
+        DRAG_DROP = "dragDropText",
+        HIGHLIGHT_MATCHES = "highlightMatches",
+        LINEWISE_COPY_CUT = "lineWiseCopyCut",
+        SCROLL_PAST_END = "scrollPastEnd",
+        SHOW_CURSOR_SELECT = "showCursorWhenSelecting",
+        SHOW_LINE_NUMBERS = "showLineNumbers",
+        SMART_INDENT = "smartIndent",
+        SOFT_TABS = "softTabs",
+        SPACE_UNITS = "spaceUnits",
+        STYLE_ACTIVE_LINE = "styleActiveLine",
+        TAB_SIZE = "tabSize",
+        UPPERCASE_COLORS = "uppercaseColors",
+        USE_TAB_CHAR = "useTabChar",
+        WORD_WRAP = "wordWrap",
+        AUTO_HIDE_SEARCH = "autoHideSearch",
         INDENT_LINE_COMMENT = "indentLineComment",
-        INPUT_STYLE         = "inputStyle";
+        INDENT_LINE_COMMENT = "indentLineComment",
+        INPUT_STYLE = "inputStyle";
```

Comment:
Most of these changes are not necessary. Just keep the values aligned as before.